admin/commands/management/remove.go

```diff
@@ -42,9 +45,53 @@ type removeMySQLCommand struct {
 	ServiceType string
 	ServiceName string
 	ServiceID   string
+	NodeID      string
```

Comment:
It seems that flag for that argument is missing

Reply:
let's remove it